src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import App from './containers/App';
+import App from './components/App';
```

Comment:
Same applies here. why did you change this ?

Reply:
Working on it as well.